mlir/unittests/IR/InterfaceTest.cpp

```diff
@@ -84,3 +87,40 @@ TEST(InterfaceTest, TestImplicitConversion) {
   typeA = typeB;
   EXPECT_EQ(typeA, typeB);
 }
+
+TEST(OperationInterfaceTest, CastOpToInterface) {
```

Comment:
```suggestion
TEST(OperationInterfaceTest, CastOpInterfaceToOpInterface) {
```

Would this name be more accurate to what this test coverage adds?